hack/test/groupsnapshot.storage.k8s.io_volumegroupsnapshots.yaml

```diff
@@ -43,7 +43,7 @@ spec:
     - jsonPath: .metadata.creationTimestamp
       name: Age
       type: date
-    name: v1alpha1
+    name: v1beta1
```

Comment:
same change is required for volumegroupsnapshotcontent CRD/yaml as well. i don't see any change for it. is this on purpose?

Reply:
We don't access `volumegroupsnapshotcontent`

Reply:
Right. I'll add it to the test directory.

Reply:
Added